bus/engineproxy.c

```diff
@@ -342,6 +384,9 @@ bus_engine_proxy_class_init (BusEngineProxyClass *class)
             bus_marshal_VOID__VOID,
             G_TYPE_NONE,
             0);
+    g_signal_set_va_marshaller (engine_signals[CURSOR_UP_LOOKUP_TABLE],
```

Comment:
I think s/CURSOR_UP_LOOKUP_TABLE/PAGE_DOWN_LOOKUP_TABLE/ ?

Reply:
Thanks! I tried so hard to not mess these up :smile: